tests/scripts/t3000/run_t3000_frequent_tests.sh

```diff
@@ -0,0 +1,121 @@
+
+#/bin/bash
+set -eo pipefail
+
+run_t3000_ethernet_tests() {
+  # Record the start time
+  start_time=$(date +%s)
+
+  echo "LOG_METAL: Running run_t3000_ethernet_tests"
+
+  pytest tests/tt_metal/microbenchmarks/ethernet/test_ethernet_bidirectional_bandwidth_microbenchmark.py
+  pytest tests/tt_metal/microbenchmarks/ethernet/test_ethernet_ring_latency_microbenchmark.py
+
+  # Record the end time
+  end_time=$(date +%s)
+  duration=$((end_time - start_time))
+  echo "LOG_METAL: run_t3000_ethernet_tests $duration seconds to complete"
+}
+
+run_t3000_llama2_70b_tests() {
+  # Record the start time
+  start_time=$(date +%s)
+
+  echo "LOG_METAL: Running run_t3000_llama2_70b_tests"
+
+  # Llama2_70b related cached files and tests (the test should parse env variables similar to these)
+  export LLAMA_CKPT_DIR=/mnt/MLPerf/tt_dnn-models/llama-2/llama-2-70b-repacked/
+  export LLAMA_TOKENIZER_PATH=/mnt/MLPerf/tt_dnn-models/llama-2/tokenizer.model
+  export LLAMA_CACHE_PATH=/mnt/MLPerf/tt_dnn-models/llama-2/llama-data-cache/weights-cache-2
+
+  pytest models/demos/t3000/llama2_70b/tests/test_llama_mlp_t3000.py
+  pytest models/demos/t3000/llama2_70b/tests/test_llama_attention_t3000.py
+  pytest models/demos/t3000/llama2_70b/tests/test_llama_decoder_t3000.py
+  pytest models/demos/t3000/llama2_70b/tests/test_llama_model_t3000.py
+
+  # Record the end time
+  end_time=$(date +%s)
+  duration=$((end_time - start_time))
+  echo "LOG_METAL: run_t3000_llama2_70b_tests $duration seconds to complete"
+}
+
+run_t3000_mistral_tests() {
```

Comment:
should be run_t3000_mixtral_tests - mistral and mixtral are different models (latter is MoE) and we might actually run mistral7b on t3000 in the future.